src/raft_log.rs

```diff
@@ -320,6 +328,11 @@ impl<T: Storage> RaftLog<T> {
                 self.persisted,
             )
         }
+        self.applied_to_unchecked(idx);
+    }
+
+    #[inline]
+    pub(crate) fn applied_to_unchecked(&mut self, idx: u64) {
```

Comment:
Seems `pub` is enough?

Reply:
This is an internal function only used for initialization, we don't want it be called by end user.